core/templates/pages/exploration-editor-page/services/version-history.service.ts

```diff
@@ -127,8 +127,10 @@ export class VersionHistoryService {
   }
 
   canShowBackwardStateDiffData(): boolean {
+    if (this.currentPositionInStateVersionHistoryList === null) {
```

Comment:
Why would it be null? This seems weird and I wonder if we can do something to avoid having to deal with this case (i.e. the variable should never be null). What does the variable mean exactly and what's a case in which it would be null?

Reply:
Actually, we can avoid this. I have removed this now. Thanks!